integration_tests/test_users_real_api.py

```diff
@@ -0,0 +1,139 @@
+import json
+import os
+
+from jcapiv1 import ApiClient, Configuration, Systemuserslist, Systemuserputpost
+
+from jccli import cli
+from click.testing import CliRunner, Result
+from jcapiv1.api.systemusers_api import SystemusersApi
+
+
+
+class TestUsersRealApi:
+    @classmethod
+    def setup_class(cls):
+        configuration = Configuration()
+        api_key = os.getenv('JC_API_KEY')
+        configuration.api_key['x-api-key'] = api_key
+        cls.systemusers_api = SystemusersApi(ApiClient(configuration=configuration))
+        cls.api_key = api_key
+
+        current_users: Systemuserslist = cls.systemusers_api.systemusers_list(
+            content_type='application/json',
+            accept='application/json'
+        )
+        assert current_users.total_count == 0
+
+    def test_create_user(self):
+        USERNAME = 'fakeuser567'
+        EMAIL = 'fakeemail3@fakesite.org'
+
+        runner = CliRunner()
+
+        result = runner.invoke(cli.cli, [
+            '--key',
+            self.api_key,
+            'user',
+            'create',
+            '--username',
+            USERNAME,
+            '--email',
+            EMAIL
+        ])
+        if result.exit_code:
+            raise ValueError(
+                "create-user exited with status code: %s;\nmessage was: %s" % (result.exit_code, result.exception)
+            )
+
+        generated_id = json.loads(result.output)['id']
+
+        # Use API to delete created user
```

Comment:
would it make sense to create a [teardown](https://docs.python.org/3/library/unittest.html#unittest.TestCase.tearDown) method to cleanup users after every test?

Reply:
Maybe worth it, but we would be adding more API calls (get user IDs -> call delete for each one) to a test suite that's already pretty slow.

Reply:
hmm, maybe we just shouldn't go overboard with these tests. remove as much duplication as possible?  for example you probably don't need  a separate `test_delete_user` since you are basically deleting users on every test already.